### PR TITLE
Update spec for ProxySQL declarative configuration and OpsReq

### PR DESCRIPTION
**Pull request by TasdidurRahman**
Signed-off-by: TasdidurRahman <tasdid@appscode.com>

### CODE REVIEW

File: apis/kubedb/v1alpha2/proxysql_types.go [hunk 1]
```diff
@@ -58,11 +59,76 @@ type ProxySQL struct {
...
 	// AutoOps contains configuration of automatic ops-request-recommendation generation
 	// +optional
 	AutoOps AutoOpsSpec `json:"autoOps,omitempty"`
 
+	// +optional
+	// SyncUsers is a boolean type and when enabled, operator fetches all users created in the backend server to the ProxySQL server
+	SyncUsers bool `json:"syncUsers,omitempty"`
+
+	// +optional
+	// InitConfiguration contains information with which the proxysql will bootstrap (only 4 fields are configurable)
```
Comment (spectro30) [inline apis/kubedb/v1alpha2/proxysql_types.go:129]: (only 4 ~~fields~~ tables are configurable)

File: apis/kubedb/v1alpha2/proxysql_types.go [hunk 2]
```diff
@@ -58,11 +59,76 @@ type ProxySQL struct {
...
 }
 
+type MySQLUser struct {
+	Username string `json:"username"`
+
+	// +optional
+	Active *int `json:"active,omitempty"`
+
+	// +optional
+	UseSsl int `json:"use_ssl,omitempty"`
```
Comment (spectro30) [inline apis/kubedb/v1alpha2/proxysql_types.go:69]: ~~UseSsl~~ UseSSL int `json:"use_ssl,omitempty"`

File: apis/kubedb/v1alpha2/proxysql_types.go (hunk 1 above)
Comment (TasdidurRahman) [reply apis/kubedb/v1alpha2/proxysql_types.go]: fixed

File: apis/kubedb/v1alpha2/proxysql_types.go (hunk 2 above)
Comment (TasdidurRahman) [reply apis/kubedb/v1alpha2/proxysql_types.go]: fixed

**Review by spectro30 — APPROVED** [review]
LGTM

**Pull request merged**